src/lib/common/Configuration.cpp

```diff
@@ -51,6 +51,7 @@ const struct config Configuration::valid_config[] = {
 	{ "slots.removable",		CONFIG_TYPE_BOOL },
 	{ "slots.mechanisms",		CONFIG_TYPE_STRING },
 	{ "library.reset_on_fork",	CONFIG_TYPE_BOOL },
+	{ "login.fail_delay",		CONFIG_TYPE_INT },
```

Comment:
I'd recommend to add a default in `softhsm2.conf.in` and `softhsm2.conf.5.in` to document this new feature as well, e.g. somewhere here:

https://github.com/softhsm/SoftHSMv2/blob/a8cabf57736e15f6a1b3ac611489826317d2126d/src/lib/common/softhsm2.conf.in#L17

https://github.com/softhsm/SoftHSMv2/blob/a8cabf57736e15f6a1b3ac611489826317d2126d/src/lib/common/softhsm2.conf.5.in#L95

Also to indicate which units (e.g. seconds or milliseconds) these are supposed to be set to.